src/main/java/org/broadinstitute/hellbender/tools/spark/sv/evidence/BreakpointEvidence.java

```diff
@@ -709,6 +881,33 @@ public DiscordantReadPairEvidence(final Kryo kryo, final Input input) {
             targetQuality = input.readInt();
         }
 
+        /**
+         * Constructor used by fromStringRep() to construct BreakpointEvidence
+         */
+        private DiscordantReadPairEvidence(final SVInterval interval, final int weight,
+                                  final String templateName, final TemplateFragmentOrdinal fragmentOrdinal,
+                                  final boolean validated, final boolean forwardStrand,
+                                  final String cigarString, final int mappingQuality,
+                                  final int templateSize, final String readGroup,
+                                  final List<StrandedInterval> distalTargets) {
+            super(interval, weight, templateName, fragmentOrdinal, validated, forwardStrand, cigarString,
+                    mappingQuality, templateSize, readGroup);
+            switch(distalTargets.size()) {
```

Comment:
In principle there could be more than one distal target. The design decision was made throughout the pipeline to keep them as a list, but for now restrict to highest quality distal target if there would otherwise be more than one.